index.js

```diff
@@ -994,29 +1030,20 @@ class GithubScm extends Scm {
      * @async  _getChangedFiles
      * @param  {Object}   config
      * @param  {String}   config.type      Can be 'pr' or 'repo'
-     * @param  {Object}   config.payload   The webhook payload received from the SCM service.
+     * @param  {Object}   [config.payload]   The webhook payload received from the SCM service.
      * @param  {String}   config.token     Service token to authenticate with Github
      * @param  {String}   [config.scmUri]  The scmUri to get PR info of
      * @param  {Integer}  [config.prNum]   The PR number
      * @return {Promise}                   Resolves to an array of filenames of the changed files
      */
     async _getChangedFiles({ type, payload, token, scmUri, prNum }) {
         if (type === 'pr') {
-            let owner;
-            let repo;
-            let number;
-
-            if (scmUri && prNum) {
```

Comment:
Don't we need to check this if statement?

Reply:
The use case of `scmUri === null, prNum === null, and payload !== null` does'nt exists.
 When `type` is `pr`, `_getChangedFiles` needs `scmUri` and `prNum` because of calling `getPrMergeable`

Reply:
I understand.